extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -640,6 +640,10 @@ export class AmpStoryPage extends AMP.BaseElement {
       } else {
         return mediaPool.play(
             /** @type {!./media-pool.DomElementDef} */ (mediaEl)).catch(() => {
+          if (mediaEl.tagName !== 'VIDEO') {
```

Comment:
would it be better if it was an optimistic condition? it seems like the next two lines are exclusive for videos

```
if (mediaEl.tagName === 'VIDEO') {
  this.debounceToggleLoadingSpinner_(false);
  this.togglePlayMessage_(true);
}
```

Reply:
Done